toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -911,6 +940,15 @@ PreferencesPopup::PreferencesPopup()
       new DVGui::IntLineEdit(this, m_pref->getDefaultTaskChunkSize(), 1, 2000);
   CheckBox *sceneNumberingCB = new CheckBox(tr("Show Info in Rendered Frames"));
 
+  m_projectRootDocuments = new CheckBox(tr("My Documents/OpenToonz*"), this);
+  m_projectRootDesktop   = new CheckBox(tr("Desktop/OpenToonz*"), this);
+  m_projectRootStuff     = new CheckBox(tr("Stuff Folder*"), this);
+  m_projectRootCustom    = new CheckBox(tr("Custom*"), this);
+  m_customProjectRootFileField = new DVGui::FileField(this, QString(""));
+  m_customProjectRootLabel     = new QLabel(tr("Custom Project Path(s): "));
+  m_projectRootDirections      = new QLabel(
+      tr("Advanced: Multiple paths can be separated by ** (No Spaces)"));
+
```

Comment:
I chose the ** because a semicolon is a valid character for a Windows path.  I don't think ** is valid in any os.  Thoughts?